collectors/proc.plugin/proc_meminfo.c

```diff
@@ -146,7 +146,7 @@ int do_proc_meminfo(int update_every, usec_t dt) {
     // --------------------------------------------------------------------
 
     // http://stackoverflow.com/questions/3019748/how-to-reliably-measure-available-memory-in-linux
```

Comment:
```suggestion
    // http://calimeroteknik.free.fr/blag/?article20/really-used-memory-on-gnu-linux
```

old one no longer valid i think